doc/nrf/protocols/bt_mesh/configuring.rst

```diff
@@ -35,11 +35,113 @@ Mesh models
 The |NCS| Bluetooth mesh model implementations are optional features, and each model has individual Kconfig options that must be explicitly enabled.
 See :ref:`bt_mesh_models` for details.
 
+Mesh settings/performance
+=========================
+
+More information about mesh memory footprint, see :ref:`app_memory_bt_mesh`.
+
+* :kconfig:option:`CONFIG_BT_MESH_PROXY_USE_DEVICE_NAME` - Includes GAP device name in scan response when the GATT Proxy feature is enabled.
+* :kconfig:option:`CONFIG_BT_MESH_DK_PROV` - Bluetooth mesh provisioning handler for the nRF5x development kits.
+* :kconfig:option:`CONFIG_BT_MESH_ADV_BUF_COUNT` - Number of advertising buffers for local messages. Increase to improve performance, at cost of increased RAM usage.
+* :kconfig:option:`CONFIG_BT_MESH_ADV_EXT_GATT_SEPARATE` - Use a separate extended advertising set for GATT Server Advertising.
```

Comment:
Shouldn't be added `CONFIG_BT_MESH_ADV_EXT_FRIEND_SEPARATE` either?

Reply:
What do you mean? Should we add CONFIG_BT_MESH_ADV_EXT_FRIEND_SEPARATE or should we remove CONFIG_BT_MESH_ADV_EXT_GATT_SEPARATE.

Reply:
add CONFIG_BT_MESH_ADV_EXT_FRIEND_SEPARATE. I didn't write anything about `CONFIG_BT_MESH_ADV_EXT_GATT_SEPARATE`.

Reply:
Ok, then I did the correct thing by adding FRIEND_SEPARATE.